docs/rest-api-spec.md

```diff
@@ -0,0 +1,178 @@
+
+# API outline
+
+baseurl: /api/v2/
```

Comment:
Is it `explorer.ooni.io/api/v2/` endpoint? Why is baseURL `api.ooni.io/api/` ?
I understand that `api.ooni.io` is CORSed, but should we merge both endpoints to single origin (domain) with nginx or not?

Reply:
This is actually a document that is stale and mostly a collection of notes to ensure I was doing the compatibility layer properly. You can delete it if you want.

Reply:
/api/v2 does not exist.